packages/web/src/views/governance/components/proposals-list/create-proposal-modal/CreateProposalModal.tsx

```diff
@@ -295,7 +314,7 @@ const CreateProposalModal: React.FC<Props> = ({
                   ))}
                 </BoxContent>
               )}
-              <BoxContent label="Deposit">
+              {/* <BoxContent label="Deposit">
```

Comment:
Is this the required code?

Reply:
This component is from legacy implements.
I think it might be required later.